.github/workflows/on-push.yml

```diff
@@ -3,6 +3,8 @@ name: Push Build and test ERDDAP
 on:
   push:
     branches: [ "main" ]
+    tags:
+      - 'v*'
```

Comment:
I have a slightly preference for:

```
  release:
    types:
      - published
```

That would force the tag to always start with a `v` and surprise us if we ever drop that in the future.

I can see people wanting to publish the docker image in a tag tough, that way the docker image info can be part of the release notes.